docker/mcp/services.py

```
@@ -53,19 +53,28 @@ def fetch_search_results(query: str, api_key: str) -> List[Dict[str, Any]]:
         'X-API-KEY': api_key,
         'Content-Type': 'application/json'
     }
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)